Makefile

```diff
@@ -19,3 +19,10 @@ package:
 .PHONY: clean
 clean:
 	rm -rf dist/ build/ kapitan.egg-info/
+
+.PHONY: pycodestyle
+pycodestyle:
+	which pycodestyle || pip3 install --user pycodestyle
```

Comment:
Some people might have pip as virtualenv, we should just print how to install it and exit

Reply:
Fixed